pyface/tests/test_toolkit.py

```diff
@@ -27,13 +33,7 @@ def test_bad_import(self):
 
     def test_core_plugins(self):
         # test that we can see appropriate core entrypoints
-        plugins = set(
-            entry_point.name
-            for entry_point in pkg_resources.iter_entry_points(
-                "pyface.toolkits"
-            )
-        )
-
+        plugins = set(ep.name for ep in entry_points()['pyface.toolkits'])
```

Comment:
We can use set comprehensions here, no need for Python 2.6 compatibility any more!